config/routes.rb

```diff
@@ -116,6 +116,10 @@
     member do
       get :schedule
     end
+
+    member do
+      get :allevents
```

Comment:
By slug he means the short_title of a conference.
And no, we don't have that route, because all routes for events go through program.
(Exception:  api_v1_conference_events_path	GET	/api/v1/conferences/:conference_id/events(.:format)	api/v1/events#index {:format=>"json"} )